pyproject.toml

```diff
@@ -15,8 +15,33 @@ dynamic = ['version']
 name = 'allowedflare'
 readme = 'README.md'
 
+[project.optional-dependencies]
+precommit = [
```

Comment:
Doesn't `pre-commit` handle its deps itself? Or is this to enable local usage of all `pre-commit` tools

Reply:
This is for system/local installation of tools like codespell which pre-commit will use.